### PR TITLE
About module fixes

### LINKED ISSUE #127: About module broken at some widths
Wide, the text in the video thumb gets cut off:

Screenshot 2019-07-22 at 14 56 01

Narrow, the text wraps too tightly:

Screenshot 2019-07-22 at 14 56 38

### PR DESCRIPTION
**Pull request by surma**
Closes #127

### CODE REVIEW

File: src/_includes/about/style.css [hunk 1]
```diff
@@ -5,3 +5,11 @@
 .about-media {
   composes: media from '../module.css';
 }
+
+.aspect-ratio {
+  composes: aspect-ratio from '../utils.css';
```
Comment (surma) [inline src/_includes/about/style.css]: Is this how utils us supposed to be used? It makes sense to me, but not sure if there’s a more convenient way.

File: src/_includes/layouts/main/style.css [hunk 2]
```diff
@@ -62,7 +62,8 @@ strong {
 }
 
 body {
-  margin: 0;
+  max-width: 1920px;
```
Comment (surma) [inline src/_includes/layouts/main/style.css]: Our designs only go up to 1920px width, and it gets kinda ridiculous with wider displays. WDYT?

Comment (jakearchibald) [reply src/_includes/layouts/main/style.css]: I don't think this works with our backgrounds (ideally they'd stretch full width and the content within would be centered).

I reckon: Revert this, and create an issue for @argyleink

File: src/_includes/about/style.css (hunk 1 above)
Comment (jakearchibald) [reply src/_includes/about/style.css]: If we're just going to use them verbatim, then you could `{% className utilCSS, 'aspect-ratio' %}`. But this works fine too (and avoids pulling in the whole utils CSS).

File: src/_includes/layouts/main/style.css (hunk 2 above)
Comment (surma) [reply src/_includes/layouts/main/style.css]: ack

File: src/_includes/about/style.css (hunk 1 above)
Comment (surma) [reply src/_includes/about/style.css]: Yeah I was worried about every module adding a <link> tag. Or we just pull in utils globally, but will it get tree-shooken?

File: src/_includes/layouts/main/style.css (hunk 2 above)
Comment (surma) [reply src/_includes/layouts/main/style.css]: Open #130

File: src/_includes/module.css [hunk 3]
```diff
@@ -12,7 +12,7 @@
     color: var(--grey);
     text-transform: lowercase;
   }
-  @media (min-width: 720px) {
+  @media (min-width: 1280px) {
```
Comment (jakearchibald) [inline src/_includes/module.css]: Keep this at 720 so it's our 'main' breakpoint?

Comment (jakearchibald) [reply src/_includes/module.css]: Ah, merged. I'll do this.

**Pull request merged**